### PR TITLE
LPS-83285 SF

### PR DESCRIPTION
**Pull request by izaera**

### CODE REVIEW

**Comment by izaera** [comment]
ci:test

File: modules/apps/npm-portlet-extender/npm-portlet-extender/build.gradle [hunk 1]
```diff
@@ -0,0 +1,10 @@
+dependencies {
+	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "default"
+	compileOnly group: "com.liferay.portal", name: "com.liferay.util.taglib", version: "default"
+	compileOnly group: "javax.portlet", name: "portlet-api", version: "2.0"
```
Comment (jbalsas) [inline modules/apps/npm-portlet-extender/npm-portlet-extender/build.gradle]: Should this be `"portlet-api", version:"3.0.0"`? That seems to be the case for `master`....

File: modules/apps/npm-portlet-extender/npm-portlet-extender/bnd.bnd [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Bundle-Name: Liferay npm Portlet Extender
+Bundle-SymbolicName: com.liferay.npm.portlet.extender
+Bundle-Version: 3.0.0
```
Comment (jbalsas) [inline modules/apps/npm-portlet-extender/npm-portlet-extender/bnd.bnd]: Should this be `1.0.0`?

File: modules/apps/npm-portlet-extender/app.bnd [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+Liferay-Releng-Bundle: false
+Liferay-Releng-Category:
+Liferay-Releng-Demo-Url:
+Liferay-Releng-Deprecated: false
+Liferay-Releng-Fix-Delivery-Method:
+Liferay-Releng-Labs: true
+Liferay-Releng-Marketplace: true
+Liferay-Releng-Portal-Required: false
+Liferay-Releng-Public: ${liferay.releng.public}
+Liferay-Releng-Restart-Required: false
```
Comment (jbalsas) [inline modules/apps/npm-portlet-extender/app.bnd]: Portal SF will force this to `true` when running `ant format-source` from `portal-impl`. We should keep it like that for now :(

File: modules/apps/npm-portlet-extender/npm-portlet-extender/bnd.bnd (hunk 2 above)
Comment (izaera) [reply modules/apps/npm-portlet-extender/npm-portlet-extender/bnd.bnd]: Yes, I think. It's maybe a copy-paste error.

File: modules/apps/npm-portlet-extender/npm-portlet-extender/build.gradle (hunk 1 above)
Comment (izaera) [reply modules/apps/npm-portlet-extender/npm-portlet-extender/build.gradle]: 👍

Comment (izaera) [reply modules/apps/npm-portlet-extender/npm-portlet-extender/build.gradle]: Though, I don't know how it worked in my machine... 🤔

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/61403. [See changes here.](https://github.com/jbalsas/liferay-portal/compare/9780e312732a~3...9780e312732a) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**